src/darwin/Framework/CHIP/MTRCommissioningParameters.h

```diff
@@ -22,40 +22,62 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MTRDeviceAttestationDelegate;
 
 /**
- * The class definition for the CHIPCommissioningParameters
- *
+ * Information that can be provided to commissionWithNodeID to commision devices.
  */
 @interface MTRCommissioningParameters : NSObject
 
 /**
- *  The CSRNonce
+ * The nonce to use when requesting a CSR for the node's operational
+ * certificate.
+ *
+ * If nil, a random nonce will be generated automatically.
+ *
+ * If not nil, must be 32 bytes of data.
  */
-@property (nonatomic, copy, nullable) NSData * CSRNonce;
+@property (nonatomic, copy, nullable) NSData * csrNonce MTR_NEWLY_AVAILABLE;
```

Comment:
I think `CSRNonce` was the correct capitalisation according to the usual Objective C style rules.

Reply:
Let me double-check this one.

Reply:
Leaving for now and we can re-evaluate when we do our second API review.